operator/bundle_helpers/README.md

```diff
@@ -1,27 +1,54 @@
 # Bundle Helpers
 
-## Python Dependency Management
-
 For hermetic builds with Konflux, we need to provide the full list of resolved dependencies in `requirements.txt`.
 These will be prefetched with Cachi2 and made available to the container image build.
 
-We use Poetry to manage Python dependencies and for generating the complete `requirements.txt`.
-This tool can generate the full dependency tree for Python dependencies.
+## Python Dependency Management
+
+You need Python>=3.9 installed.
+We use Poetry to manage Python dependencies.
```

Comment:
We can ditch `poetry` altogether and edit `pyproject.toml` by hand. We don't need `poetry.lock` because it does not participate in the following commands and it can be deleted.

Furthermore, since `pip-compile` can take `requirments.txt` format on the input, we can replace `pyproject.toml` with `requirements-konflux.in` like this:
```
PyYAML==6.0.2
```
That's it. We don't need `pytest` in Konflux context because `pytest` command is called by the `Makefile` which we bypass.

To bypass builder image update (for now), we can introduce `requirements-upstream.txt` with
```
PyYAML==6.0
pytest==7.0.1
```

To recap:
1. Let's please get rid of `poetry` mentions in this readme.
2. Please delete `poetry.lock`.
3. (optionally) Please switch from `pyproject.toml` to `requirements.txt` format (if that works).
4. (optionally) Create a separate `requirements.txt` to unfail GHA builds.